code/modules/projectiles/projectile/bullets/revolver.dm

```diff
@@ -106,6 +106,14 @@
 		else
 			to_chat(M, "<span class='userdanger'>You get shot with the finger gun!</span>")
 
+/obj/item/projectile/bullet/c38/mime/lethal
+	damage = 20
+
+/obj/item/projectile/bullet/c38/mime/lethal/on_hit(atom/target, blocked)
+	if(iscarbon(target))
```

Comment:
You don't need to run parent here?

Reply:
Running parent would run the non lethal mime bullet proc.

I ripped this from the original mime gun projectile proc.

Reply:
See comment in file view

Reply:
on hit handles calculations and other stuff, you need to run it